rbind_cluster_county_test.R

```diff
@@ -10,14 +10,14 @@ library(sqldf)
 library(dplyr)
 registerDoMC(21)
 
-#source here indicates that we can reuse some functions that were written in other script
-source("/gpfs/data1/cmongp/lansong/cec_lan/cec_utils.R")# some functions will be used in this script
-source("/gpfs/data1/cmongp/lansong/cec_lan/config.R")#for set up the working directory
-source("/gpfs/data1/cmongp/lansong/cec_lan/forest_type.R")# useless
-source("/gpfs/data1/cmongp/lansong/cec_lan/data.R")# useless
-source("/gpfs/data1/cmongp/lansong/cec_lan/cluster_results_new.R")#useless
+# source here indicates that we can reuse some functions that were written in other scripts stored in the shared folder.
+source("shared/cec_utils.R")# some functions will be used in this script
+source("shared/config.R")# set up the working directory and expose root_dir and data_dir variables
+source("shared/forest_type.R")# useless
+source("shared/data.R")# useless
+source("shared/cluster_results_new.R")#useless
 
-# loads F3 forest, elevation, sit raster,stack all F3 forest data with sit data,and chwrite.csv(final_csv,"/gpfs/data1/cmongp/lansong/cec_lan/Sierra_Navada_Clusters/final_csvs/county_csv/Sorted_Sierra_Nevada.csv")ange the variable name to simple name
+# loads F3 forest, elevation, sit raster,stack all F3 forest data with sit data,and write final_csv to Sorted_Sierra_Nevada.csv") and change the variable name to simple name
 load_forest_data <- function(){
   log("loading forest data")
   fl <<- list.files(pattern=glob2rx("Total*.tif$")) # list all the F3 Data  There are total of 60 files 
```

Comment:
I'm not sure if you need to add a path within the list.files(path="",patter=glob2rx("Total*.tif$")) to access all tif files